src/ApiCompat/Microsoft.DotNet.ApiCompatibility/Abstractions/ElementContainer.cs

```diff
@@ -9,17 +9,17 @@ namespace Microsoft.DotNet.ApiCompatibility.Abstractions
     /// Class to wrap an Element of T with it's <see cref="MetadataInformation"/>.
     /// </summary>
     /// <typeparam name="T">The type of the Element that is holded</typeparam>
-    public class ElementContainer<T> where T : ISymbol
+    public readonly struct ElementContainer<T> where T : ISymbol
```

Comment:
Changes in this file are unrelated but are an optimization (using a `readonly struct` over `class` reduces run time costs) that I felt isn't worth putting into a separate PR.